### ISSUE TITLE
Cannot Reproduce Fraxtil Step Placement Result Using Pretrained Model

### ISSUE THREAD

**Issue by chaonan99** [chrisdonahue/ddc#10]
Hi Chris,

Thanks for this great work. I was trying to reproduce the reported result with the pretrained model provided in the `infer` directory. I used Docker so I suppose there is no inconsistency in the environment. I run the model on Fraxtil test set and saved the `placed_times` for each chart, then used the predicted times and ground truth times to compute F1-c. But I can only get ~**0.587** out of that, which is far from the **0.681** reported in your paper, and even worse than LogReg baseline. Is this model actually trained on ITG dataset, or did I do something wrong?

By the way, the link to download ITG dataset is broken. Could you please point me the way to download the dataset?

Thanks!
